eth/protocols/eth/peer.go

```diff
@@ -545,3 +562,30 @@ func (p *Peer) RequestTxs(hashes []common.Hash) error {
 	}
 	return p2p.Send(p.rw, GetPooledTransactionsMsg, GetPooledTransactionsPacket(hashes))
 }
+
+// RequestOnePendingEtx fetches a pendingEtx for a given block hash from a remote node.
+func (p *Peer) RequestOnePendingEtxs(hash common.Hash) error {
+	p.Log().Debug("Fetching a pending etx", "hash", hash)
+	if p.Version() >= ETH66 {
```

Comment:
Same issue here with a branch for 65 and 66

Reply:
removed eth65